sdk/clients/csharp/protobuf-client/src/Azure.Messaging.WebPubSub.Client.Protobuf.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.Messaging.WebPubSub.Client.Protobuf</PackageId>
+    <Description>Client Protobuf Protocol for Azure WebPubSub</Description>
+    <PackageTags>azure;webpubsub.client</PackageTags>
+    <NoWarn>$(NoWarn);0067</NoWarn>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <None Remove="Protos\webpubsub.client.proto" />
```

Comment:
Why we have such `Remove` item here?

Reply:
`webpubsub.client.proto` is only required at build time for code generation by the Protobuf library, so we have this `Remove` item to make sure it's not included in the output directory (since it's not needed at runtime).